brigade/static/scss/molecules/_card.scss

```diff
@@ -132,3 +132,20 @@ a.card {
     }
   }
 }
+
+/* Multimedia cards prominently feature a picture or video above the title */
+.card--multimedia__title {
```

Comment:
This works, but depending on how closely we want to follow BEM as a convention, a more typical naming pattern would be .`card--multimedia .card__title { ... }`
![image](https://user-images.githubusercontent.com/8628090/40393583-e38f9170-5dd4-11e8-96b6-f6923c274f86.png)